torchtext/experimental/vectors.py

```diff
@@ -0,0 +1,188 @@
+import csv
+import logging
+import os
+
+import torch
+from torch import Tensor
+import torch.nn as nn
+from tqdm import tqdm
+
+from torchtext.utils import (
+    download_from_url,
+    extract_archive
+)
+
+logger = logging.getLogger(__name__)
+
+
+def _load_token_and_vectors_from_file(file_path):
+    stoi, tokens, vectors, dup_tokens = {}, [], [], []
+    with open(file_path, 'rb') as f:
+        for line in tqdm(f):
+            token, entries = line.rstrip().split(b" ", 1)
+            vector = torch.tensor([float(c) for c in entries.split(b" ")], dtype=torch.float)
+
+            try:
+                if isinstance(token, bytes):
+                    token = token.decode('utf-8')
+            except UnicodeDecodeError:
+                logger.info("Skipping non-UTF8 token {}".format(repr(token)))
+                continue
+
+            if token in stoi:
+                logger.info("Found duplicate entries for token {} on line {}.".format(repr(token), len(vectors) + 1))
+                dup_tokens.append(token)
+                continue
+
+            stoi[token] = len(vectors)
+            tokens.append(token)
+            vectors.append(vector)
+
+    return tokens, vectors, dup_tokens
+
+
+def fast_text(language="en", unk_tensor=None, root='.data'):
```

Comment:
Should we be consistent with the legacy code for the pre-trained vector names? like FastText, GloVe.

Reply:
I think I was adhering to the Python snake_case formatting for function names. In the legacy code we named it using PascalCase because thats how classes are usually formatted. Do you think it still makes sense to change the naming even though this has become a function rather than a class?